{"name": "测试用例描述：错误的验证码登录失败", "status": "passed", "description": "测试描述：错误的验证码登录失败", "steps": [{"name": "输入错误的验证码", "status": "passed", "attachments": [{"name": "错误的验证码", "source": "f7af87f6-4f7e-4133-a834-62f34c3e43ed-attachment.attach"}], "start": 1651990813389, "stop": 1651990813389}], "attachments": [{"name": "截图", "source": "f0e45fa2-4c18-42de-9743-50566138b66b-attachment.png", "type": "image/png"}], "start": 1651990813389, "stop": 1651990814355, "uuid": "509f2b47-38de-48a1-9872-968f93413b0c", "historyId": "6c05533b4213c6c9c25514ab7813de92", "testCaseId": "638baadf0fe36cf4b1b76169efb70edf", "fullName": "test_login.Test#test_23", "labels": [{"name": "story", "value": "登录"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "epic", "value": "xxx测试"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "11376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}